source/npm/util.js

```diff
@@ -105,7 +105,18 @@ exports.checkIgnoreStrategy = ({files}) => {
 
 	if (!files && !npmignoreExists) {
 		console.log(`
-		\n${chalk.bold.yellow('Warning:')} No ${chalk.bold.cyan('files')} field specified in ${chalk.bold.magenta('package.json')} nor is a ${chalk.bold.magenta('.npmignore')} file present. Having one of those will prevent you from accidentally publishing development-specific files along with your package's source code to npm. 
+		\n${chalk.bold.yellow('Warning:')} No ${chalk.bold.cyan('files')} field specified in ${chalk.bold.magenta('package.json')} nor is a ${chalk.bold.magenta('.npmignore')} file present. Having one of those will prevent you from accidentally publishing development-specific files along with your package's source code to npm.
 		`);
 	}
 };
+
+exports.getRegistryUrl = async (pkgManager, pkg) => {
+	const args = ['config', 'get', 'registry'];
+	if (exports.isExternalRegistry(pkg)) {
```

Comment:
If this is the case, then you already know the registry URL ahead of time, and so you should return it (instead of calling `${pkgManager} config get` unnecessarily).

Reply:
@TiagoDanin Could you take a look at this?